vmdb/app/models/container_replication_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ContainerReplicationController < ActiveRecord::Base
```

Comment:
@zeari @abonas Is there a parallel in Amazon Container Services?  If so, does this class/table name reflect that?